### ISSUE TITLE
qmake failure with >= Qt 5.10

### ISSUE THREAD

**Issue by ilovezfs** [Tarsnap/tarsnap-gui#171]
```
Project ERROR: Tarsnap-gui requires Qt 5.2 or higher.
```

This is caused by https://github.com/Tarsnap/tarsnap-gui/blob/9bc8252b9e8178d9ec69594ef56f5440bdac2f6b/Tarsnap.pro#L2

```
lessThan(QT_VERSION, 5.2): error("Tarsnap-gui requires Qt 5.2 or higher.")
```

5.10 is considered to be < 5.2 since the comparison is merely numerical.

https://jenkins.brew.sh/job/Homebrew%20Testing/388/version=high_sierra/console
(duplicate since that will eventually be deleted: https://gist.github.com/ilovezfs/ac3f7495c7355af84aabc8dff8f2072a)

**Comment by gperciva** [comment]
Wow, this looks like a mess.  The latest qmake contains `versionAtLeast()`, but that appears to have been added in 5.10 or 5.11.  (It also doesn't specify whether the comparison is numerical or strings!)

I'm working on it.

**Comment by gperciva** [comment]
I think this is fixed in https://github.com/Tarsnap/tarsnap-gui/pull/172 (merged into `master` now).  Thanks for the report, @ilovezfs!

**Comment by ilovezfs** [comment]
@gperciva Yup, it's fixed! Thank you :)

**Comment by apjanke** [comment]
Since this is fixed, would you mind cutting a new 1.0.1 release to pick this up? Building using the unmodified https://github.com/Tarsnap/tarsnap-gui/archive/v1.0.tar.gz release is still failing against newer versions of Qt.

**Comment by gperciva** [comment]
Good point!  I wanted to get a few more tests in place before trying a new release, but it would be worthwhile getting this out sooner rather than later.  I'll aim for Sunday.
